build/Install-DotNetCoreSDK.ps1

```diff
@@ -0,0 +1,5 @@
+﻿$globalJson = Get-Content "..\global.json" | ConvertFrom-Json
```

Comment:
This needs to be the relative path of the script root. For example, if you run it from the base of the repo `.\build\install-dotnetcoresdk.ps1` it fails it's looking for global.json in the parent of my current directory

Reply:
Fixed on latest commit.